### PR TITLE
LPS-197821 Show new My Saved Content menu on the personal Menu

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
image

This is the first PR 
MTBD

- LPS-197821 create new web module for saved content entries
- LPS-197821 Create portlet and add the update to pass it to the version 1.0.0
- LPS-197821 show new option on personal menu
- LPS-197821 FF
- LPS-197821 buildLang

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
I do not understand about the FF...

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

File: modules/apps/saved-content/saved-content-web/src/main/java/com/liferay/saved/content/web/internal/portal/profile/ModulePortalProfile.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	@Activate
+	protected void activate(ComponentContext componentContext) {
+		init(
+			componentContext,
+			Arrays.asList(
+				PortalProfile.PORTAL_PROFILE_NAME_CE,
+				PortalProfile.PORTAL_PROFILE_NAME_DXP),
+			MySavedContentPortlet.class.getName());
+	}
+
```
Comment (adolfopa) [inline modules/apps/saved-content/saved-content-web/src/main/java/com/liferay/saved/content/web/internal/portal/profile/ModulePortalProfile.java:33]: (question) Why do we need this? It is the first time I've seen this in a module :thinking:

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**